src/web/client/extension.ts

```diff
@@ -268,12 +269,95 @@ export function processWillSaveDocument(context: vscode.ExtensionContext) {
     );
 }
 
-export function processWillStartCollaboartion() {
+export function processWillStartCollaboartion(context: vscode.ExtensionContext) {
     if (isCoPresenceEnabled()) {
```

Comment:
add a false && isCoPresenceEnabled() here for now - till we test and certify that everything is working fine

Reply:
Done: https://github.com/microsoft/powerplatform-vscode/pull/745/commits/a2922d0493b6a814573bb32a5f9add77e10aa360

Reply:
updated: https://github.com/microsoft/powerplatform-vscode/pull/745/commits/b4e93191a91d9033a6271324d1aea945fc27cccd